keanu-project/src/main/java/io/improbable/keanu/vertices/Probabilistic.java

```diff
@@ -0,0 +1,50 @@
+package io.improbable.keanu.vertices;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.collect.ImmutableList;
+
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+
+public interface Probabilistic<T> extends Observable<T> {
+
+    /**
+     * This is the natural log of the probability at the supplied value. In the
```

Comment:
I think here it'll be more clear to say that `logProb` is the natural log of pmf for discrete vertices and pdf for continuous.

Reply:
@return The natural log of the probability function at the supplied value.
     For continuous variables this is called the PDF (probability density function).
     For discrete variables this is called the PMF (probability mass function).